### PR TITLE
fix hello_world.md

### PR DESCRIPTION
**Pull request by tinosulzer**

### CODE REVIEW

File: getting_started/hello_world.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ content:
 
 ```bash
 #!/bin/bash
-#SBATCH --cpus=1
+#SBATCH --N=1
```
Comment (abillscmu) [inline getting_started/hello_world.md:25]: `--N` is not a valid slurm option. The correct syntax would be `--nodes` or `-N`. However, I think we should leave it as `-n`. `-n` specifies the number of tasks, and one can add `--cpus-per-task` or `-c` in order to specify the number of cpus for each task. Slurm has a default of 1 cpu per task, so just specifying `-n` here is valid.

File: getting_started/hello_world.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ echo "Hello World!"
 
 1) `#!/bin/bash` This is a [shebang] it tells Linux how to handle the file.
 Here's we're saying use `bash` to run this file
-2) `#SBATCH --cpus=1` This is a SLURM directive asking for 1 CPU
+2) `#SBATCH --N=1` This is a SLURM directive asking for 1 CPU
```
Comment (abillscmu) [inline getting_started/hello_world.md:34]: see above

File: getting_started/hello_world.md [hunk 3]
```diff
@@ -95,6 +95,7 @@ Hello World!
 > `#SBATCH -output` directives or `sbatch`'s command line flags `--output`.
 > See [sbatch] for more information.
 
+[sbatch]: https://slurm.schedmd.com/sbatch.html
```
Comment (abillscmu) [inline getting_started/hello_world.md:98]: thanks

**Review by abillscmu — CHANGES_REQUESTED** [review]

**Review by abillscmu — APPROVED** [review]

File: getting_started/hello_world.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ echo "Hello World!"
 
 1) `#!/bin/bash` This is a [shebang] it tells Linux how to handle the file.
 Here's we're saying use `bash` to run this file
-2) `#SBATCH --cpus=1` This is a SLURM directive asking for 1 CPU
+2) `#SBATCH -n=1` This is a SLURM directive asking for 1 CPU
```
Comment (awadell1) [inline getting_started/hello_world.md:34]: `-n` or `--ntasks` sets the maximum number of task that the batch job will launch
`--cpus` no longer looks to be a flag

Maybe `--ntasks=1` and `--cpus-per-task=1` to keep it similar to the original intent

**Review by awadell1 — CHANGES_REQUESTED** [review]

File: getting_started/hello_world.md (hunk 4 above)
Comment (abillscmu) [reply getting_started/hello_world.md:34]: from slurm docs:

```-c, --cpus-per-task=<ncpus>
Advise the Slurm controller that ensuing job steps will require ncpus number of processors per task. Without this option, the controller will just try to allocate one processor per task.```

We only have 2 options in this bash script. We're already assuming default time, account, partition, no tmp space, etc. So at some point we have to decide what defaults we want to use and those we do not. By specifying `-c`, we're simply overriding a default option with a default. If you want to change something here, just add `-c 1` but again, its just making explicit a default.

Now, I didn't notice that the syntax is still wrong here @tinosulzer when I approved this. In slurm, two dashes needs an equals and one doesn't. So pls change that.

File: getting_started/hello_world.md [hunk 5]
```diff
@@ -31,7 +31,7 @@ echo "Hello World!"
 
 1) `#!/bin/bash` This is a [shebang] it tells Linux how to handle the file.
 Here's we're saying use `bash` to run this file
-2) `#SBATCH -n=1` This is a SLURM directive asking for 1 CPU
+2) `#SBATCH -n 1` This is a SLURM directive asking for 1 task
 3) `#SBATCH --mem=2G` Now we're asking for 2G of Memory in bytes. Notice the
```
Comment (abillscmu) [inline getting_started/hello_world.md:35]: fix units and capitalization

**Review by abillscmu — CHANGES_REQUESTED** [review]

**Review by awadell1 — APPROVED** [review]
lgtm! Thanks for putting this together

**Review by abillscmu — APPROVED** [review]

**Pull request merged**